charts/karavi-observability/templates/karavi-metrics-powerscale.yaml

```diff
@@ -61,6 +67,37 @@ spec:
           readOnly: true
         - name: karavi-metrics-powerscale-configmap
           mountPath: /etc/config
+      {{- if hasKey .Values.karaviMetricsPowerscale "authorization" }}
+      {{- if eq .Values.karaviMetricsPowerscale.authorization.enabled true }}
+      - name: karavi-authorization-proxy
+        imagePullPolicy: IfNotPresent
+        image: {{ required "Must provide the authorization sidecar container image." .Values.karaviMetricsPowerscale.authorization.sidecarProxyImage }}
+        env:
+          - name: PROXY_HOST
+            value: "{{ .Values.karaviMetricsPowerscale.authorization.proxyHost }}"
+          - name: INSECURE
```

Comment:
Please update `INSECURE ` to `SKIP_CERTIFICATE_VALIDATION` to align with the new Authorization sidecar proxy nomenclature

Reply:
@shaynafinocchiaro  updated powerflex and powerscale. 
and what version sidecar can I used for testing

Reply:
You can use the nightly tag for testing purposes